spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
-
-[cofog]: http://data.okfn.org/data/core/cofog/
-
-Content requirements will necessarily vary across the different types of fiscal data. For example, the data describing high level budgets may be different from that describing day-to-day expenditures, and expenditure information may be different from revenue. Thus, our framework will distinguish different types of fiscal data.
-
-We also emphasize that what we provide is a framework rather than a strict standard. That is, we provide recommendations on what information should be provided rather than strict requirements.
-
-Finally, our recommendations place requirements on the "logical" model not the physical model. Of course, the logical model data is sourced from the physical model so requirements on the logical model ultimately place requirements on the physical model. However, by defining our requirements on the logical model, we keep the flexibility in naming and structure of the raw, source data - for example, whilst a classification dimension with COFOG data should be named `cofog` and have an attribute called `code` your source CSV could have that COFOG data in a column called "COFOG-Code" or "Classification" or any other name.
-
 
-## Required data (all categories)
+# Dimension types
 
-All datasets MUST have at least one measure. Essentially this is requiring each dataset have at least one field / column which corresponds to an "amount" of money.
+This section provides guidance, and certain requirements, on the naming of dimension attributes. For example, if you have a column representing the name of an organisation, it should be modelled as a `title` attribute on an dimension with `dimensionType` of `entity`.
 
-## Special Dimensions
+## Classification
 
-### Classifications
-
-It is common for fiscal data to be classified in various ways. A classification is a labelling of a given item with a reference to standardized codesheet.
-
-Classifications will be represented in the model as a dimension. Each classification dimension `MUST` have a `code` attribute whose value will correspond to the classification code in the official codesheet. Sometimes classifications can change and we recommend utilizing a `version` attribute if there is a need to indicate the version of a classification.
-
-Whenever we have a code attribute in a classification dimension, the licit values for that attribute consist of the numerical codes from the appropriate codesheet, with hierarchical levels separated by periods. `1.1.4.1.3` is a licit value for a dimension named `gfsm`, for example, corresponding to the code for "Turnover and other general taxes on goods and services".
-
-Classifications are of different types. The type of the classification `MAY` be indicated using the `classificationType` attribute on the dimension. Values are:
-
-* `functional`
-* `administrative`
-* `economic`
-
-#### Hierarchical Classifications
```

Comment:
I get that this has just merged in to the example. I think that is fine - any thoughts @akariv ?